### PR TITLE
guidance: new package

### PR DESCRIPTION
**Pull request by akthoma**
Adding new package guidance

### CODE REVIEW

File: var/spack/repos/builtin/packages/guidance/package.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            files = glob.iglob('*.pl')
+            for file in files:
+                perl = FileFilter(file)
+                perl.filter('#!/usr/bin/perl -w', '#!/usr/bin/env perl')
+        with working_dir(join_path('www',
+                                   'bioSequence_scripts_and_constants')):
+            files = glob.iglob('*.pl')
+            for file in files:
+                perl = FileFilter(file)
+                perl.filter('#!/usr/bin/perl -w', '#!/usr/bin/env perl')
```
Comment (junghans) [inline var/spack/repos/builtin/packages/guidance/package.py]: Could you put this in double loop?

Comment (junghans) [reply var/spack/repos/builtin/packages/guidance/package.py]:
```
for dir in Guidance Selecton and bioSequence_scripts_and_constants; do
  bla
```

Comment (akthoma) [reply var/spack/repos/builtin/packages/guidance/package.py]: In theory, I feel like this should work:
```
    def edit(self, spec, prefix):
        with working_dir('www'):
            for dir in 'Guidance', 'Selecton', 'bioSequence_scripts_and_constants':
                files = glob.iglob('*.pl')
                for file in files:
                    perl = FileFilter(file)
                    perl.filter('#!/usr/bin/perl -w', '#!/usr/bin/env perl')
```
but it doesn't--the installed files are not edited and they still have `#!/urs/bin/perl -w` at the top.

Comment (akthoma) [reply var/spack/repos/builtin/packages/guidance/package.py]: Thanks!

**Comment by akthoma** [comment]
bump

File: var/spack/repos/builtin/packages/guidance/package.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+import glob
+
+
+class Guidance(MakefilePackage):
+    """Guidance: Accurate detection of unreliable alignment regious accounting
```
Comment (hartzell) [inline var/spack/repos/builtin/packages/guidance/package.py]: regious -> regions

**Pull request merged**